conda/install.py

```diff
@@ -635,8 +648,21 @@ def messages(prefix):
     finally:
         rm_rf(path)
 
+
+def ensure_write(prefix, dist):
+    meta = load_meta(prefix, dist)
+    if not can_open_all_files_in_prefix(prefix, meta["files"]):
+        # TODO Should eventually return rather than exiting
+        sys.exit(
+            "Unable to remove files for package {pkg_name}.  Please\n"
+            "close all running processes and try again.".format(
```

Comment:
Just realized this might be confusing to inexperienced users. Maybe "close all processes running {pkg_name}" or something like that.